examples/text_labels_and_annotations/shaded_table.py

```diff
@@ -0,0 +1,73 @@
+from matplotlib import pyplot as plt
```

Comment:
latest fix changes the header.
also fixed the example to save a png rather than show the plt -- per contour.
guessing at the file name to save as...